pages/routines.js

```diff
@@ -0,0 +1,15 @@
+import {ExercisesHeader} from "../components/Exercises/ExercisesHeader";
+import Navigation from "../components/Navigation";
+import {WorkoutHeader} from "../components/Routines/RoutinesHeader";
+
+const routines = () => {
+  return (
+    <>
+      <WorkoutHeader />
+      <ExercisesHeader />
+      <Navigation />;
+    </>
+  );
```

Comment:
You could move your `<Navigation/>` component to `_app.js` instead of repeating it on every page.
This should not block the PR, but maybe you can consider this for a future refactoring-PR